pkg/output/card.go

```diff
@@ -38,15 +37,15 @@ type cardColumns struct {
 	Value interface{}
 }
 
-func PrintCards(c *cli.Context, w io.Writer, items []interface{}, opts *PrintOptions) {
+func PrintCards(c *cli.Context, w io.Writer, items []interface{}, opts *PrintOptions) error {
 	fields := opts.Fields
 	if fields == nil {
 		fields = extractFieldNames(items[0], []string{}, "", fieldsDepth)
 	}
 
 	valuesList, err := extractFieldValues(items, fields)
 	if err != nil {
-		process.ErrorAndExit("unable to extract values", err)
+		return fmt.Errorf("unable to print card view: %s", err)
```

Comment:
This is a great change! Can you use `%w` everywhere when wrapping error. It is ok to use `%v` in `tctl` itself but all libs should use `%w`. And never use `%s` for errors.